docs/get-started.md

```diff
@@ -1,170 +1,64 @@
-Getting Started
-===============
-
-In order to consume these components you will require a system that
-utilised a bundler (such as [Webpack]) that can process imported assets
-such as images, fonts and [Sass] files. [Create React App] may be able to
-do this but does not provide Server-Side Rendering (SSR). You will also
-need to ensure that you provide an instance of [react-router].
-
-As such, we suggest that you use our specially designed tech stack for
-this purpose. You can start a brand new project or prototype using the
-following steps.
+Prototyping
+===========
 
+This guide explains how to create prototypes using the Home Office Design System.
 
 ### 1. Set up your repository
 
-You should first set up a blank code repository in the provider of your
-choice. For the purposes of this document I will assume [GitHub] as we
-provide extra CI support through [GitHub Actions].
+Set up a blank code repository in [GitHub](https://github.com/) as
+extra CI support is provided through [GitHub Actions](https://github.com/features/actions).
 
 
 ### 2. Run the following commands locally
 
-(The parts in ALL CAPS should be replaced with something specific to
-you.)
+Using terminal, run the following commands below (the parts in ALL CAPS should be replaced with something specific to
+you):
 
 ```shell
-mkdir YOUR-PROJECT
-cd YOUR-PROJECT
+mkdir YOUR-PROTOTYPE
+cd YOUR-PROTOTYPE
 git init
-git remote add origin git@github.com:YOUR_USER/YOUR-PROJECT.git
-npm init @hods
+git remote add origin git@github.com:YOUR_USER/YOUR-PROTOTYPE.git
```

Comment:
If we eliminate section one, we can delete this line.

Reply:
Removed and added this line under Step 5: Commit and push your new prototype in apps/docs/src/common/pages/get-started/prototype.tsx in this  [PR #57](https://github.com/UKHomeOffice/hods-poc/pull/57)